Documentation for license registry validation (#104)
dotnet-project-file-analyzers/dotnet-project-file-analyzers.github.io: rules/Proj0507.md

---
parent: Licensing
ancestor: Rules
---

> # Proj0507: Third-party license registry must be unconditiona

Reviewer: ```suggestion
# Proj0507: Third-party license registry must be unconditional
```

Using a [NuGet](https://www.nuget.org) package implies that you
and/or your company explicitly agree with the legally binding conditions of the
license and the copyright of the owner of the package.

> The .NET project file analyezers can not execute all conditions while

Reviewer: ```suggestion
The .NET project file analyzers can not execute all conditions while
```

> interpeting the MS-Build files. Therefor, this explicitly disallowed.

Reviewer: ```suggestion
interpreting the MS-Build files. Therefor, this is explicitly disallowed.
```
